appcues/src/main/java/com/appcues/data/mapper/experience/ExperienceMapper.kt

```diff
@@ -51,23 +51,23 @@ internal class ExperienceMapper(
     }
 
     private fun List<StepContainerResponse>.mapToStepContainer(
-        superTraits: List<TraitResponse>,
+        superTraits: List<ExperienceTrait>,
         superActions: Map<UUID, List<ActionResponse>>?
     ) = map { stepContainerResponse ->
-        val mergedTraits = stepContainerResponse.traits.mergeTraits(superTraits)
-        val mappedTraits = traitsMapper.map(mergedTraits)
+        val mappedTraits = traitsMapper.map(stepContainerResponse.traits)
+        val mergedTraits = mappedTraits.mergeTraits(superTraits, EXPERIENCE)
```

Comment:
here we are merging traits in this node with traits  where level > EXPERIENCE, meaning GROUP, and STEP 

**looking back now I it would be nice to change so its level >= GROUP